src/diffpy/utils/diffraction_objects.py

```diff
@@ -218,6 +220,21 @@ def __add__(self, other):
     __radd__ = __add__
 
     def __sub__(self, other):
+        """Subtract scalar value or another DiffractionObject to the yarray of
+        the DiffractionObject.
+
+        This method behaves similarly to the `__add__` method, but performs subtraction instead of addition.
```

Comment:
Since __sub__, __mul__, and __truediv__  pretty much have the same docstrings, I asked them to refer to __mul__ while the examples are still unique.